### ISSUE TITLE
[Feature Request] Pass CancellationToken through method without CallContext

### ISSUE THREAD

**Issue by ElderJames** [protobuf-net/protobuf-net.Grpc#43]
The method  in service interface like
```
 IAsyncEnumerable<TimeResult> SubscribeAsync(CancellationToken token);
```
When it cancel on the client, the server can get it.
Can you construct a CallContext with this parameter in the proxy implementation?

**Comment by mgravell** [comment]
Interesting. I like it. I'll need to assess impact, but conceptually :
makes sense.

**Comment by mgravell** [comment]
This is fully implemented from 1.0.81 onwards (fixed in #95)
